src/Tizen.Multimedia.Recorder/Recorder/VideoRecorder.Capabilities.cs

```diff
@@ -49,6 +49,11 @@ private static IEnumerable<Size> LoadVideoResolutions(CameraDevice device)
         {
             using (var camera = new Camera(device))
             {
+                if (!Features.IsSupported(RecorderFeatures.VideoRecorder))
```

Comment:
I think if the feature is not supported, the camera doesn't need to be created which means checking needs to be first followed by camera creation.
And I wonder it is possilbe that camera can be created when the video recorder feature is false, if so, checking video recorder feature after camera creation is meaningless.

Reply:
I'll update it to check video_recording feature first.